air/src/execution_step/instructions/mod.rs

```diff
@@ -47,20 +47,24 @@ use crate::execution_step::TraceHandler;
 
 use air_parser::ast::Instruction;
 
-// TODO: move all error set logic from macros into the execution context
+/// Executes am instruction and updates both errors if needed.
```

Comment:
"an instruction". We speak English, not Portuguese!

Reply:
My speak gud Anglish.

Reply:
Fixed